pom.xml

```diff
@@ -1071,6 +1071,13 @@
                 <version>${netty4.version}</version>
             </dependency>
             
+            <!-- set version for snakeyaml transitive dependencies -->
```

Comment:
If we add a dependencyManagement entry, then we ought to remove the version from the dependency.